FusionIIIT/applications/ps1/api/views.py

```diff
@@ -126,7 +140,7 @@ def indentView(request, id):
 def draftView(request, id):
     if request.method == 'GET':
         des = HoldsDesignation.objects.all().select_related().filter(user=request.user).first()
-        if request.session.get('currentDesignationSelected') == "student":
```

Comment:
could you please tell why you have remove this?
Like here we need designation from dashboard's dropdown.

Reply:
it was earlier added by mistake , we can't access the current selected designation using this for APIs though it can be used for web version.

Reply:
As by just using the token we receive on login it is not possible to access the currently selected designation of the user we take help of a id param which helps us in obtaining so using a Table named **HoldsDesignation** .